src/amo/components/AddonMoreInfo/index.js

```diff
@@ -278,20 +291,37 @@ export class AddonMoreInfoBase extends React.Component<InternalProps> {
 }
 
 export const mapStateToProps = (state: AppState, ownProps: Props) => {
-  const { addon } = ownProps;
+  const { addon, i18n } = ownProps;
+  let currentVersion = null;
+  let versionInfo = null;
+
+  if (addon && addon.currentVersionId) {
+    currentVersion = getVersionById({
+      id: addon.currentVersionId,
+      state: state.versions,
+    });
+  }
+
+  if (currentVersion) {
+    versionInfo = getVersionInfo({
+      i18n,
+      state: state.versions,
+      userAgentInfo: state.api.userAgentInfo,
+      versionId: currentVersion.id,
+    });
+  }
+
   return {
+    currentVersion,
+    versionInfo,
     hasStatsPermission: hasPermission(state, STATS_VIEW),
     userId: state.users.currentUserID,
-    currentVersion:
-      addon && addon.currentVersionId
-        ? getVersionById({ id: addon.currentVersionId, state: state.versions })
-        : null,
   };
 };
 
 const AddonMoreInfo: React.ComponentType<Props> = compose(
-  connect(mapStateToProps),
   translate(),
+  connect(mapStateToProps),
```

Comment:
Why is this needed?

Reply:
I changed the order because `getVersionInfo` inside `mapStateToProps` needs `i18n` prop.

Reply:
ah, I see. Thanks!